packages/model-viewer/src/test/features/scene-graph-spec.ts

```diff
@@ -74,6 +75,31 @@ suite('ModelViewerElementBase with SceneGraphMixin', () => {
         expect(exported).to.be.not.undefined;
         expect(exported.size).to.be.greaterThan(500);
       });
+
+      test('has variants', () => {
+        expect(element[$scene].currentGLTF!.userData.variants.length)
+            .to.be.eq(3);
+        const glTFroot = element[$scene].modelContainer.children[0];
+        expect(glTFroot.children[0].userData.variantMaterials.size).to.be.eq(3);
+        expect(glTFroot.children[1].userData.variantMaterials.size).to.be.eq(3);
+      });
+
+      test('exports and reimports the model with variants', async () => {
+        const exported = await element.exportScene({binary: true});
+        const url = URL.createObjectURL(exported);
+        element.src = url;
+        await waitForEvent(element, 'load');
+        await rafPasses();
+
+        expect(element[$scene].currentGLTF!.userData.variants.length)
+            .to.be.eq(3);
+        // TODO: export is putting in an extra node layer, because the loader
+        // gives us a Group, but if the exporter doesn't get a Scene, then it
+        // wraps everything in an "AuxScene" node. Feels like a three.js bug.
+        const glTFroot = element[$scene].modelContainer.children[0].children[0];
```

Comment:
@mrdoob This comment is for you. I can work around it, but it's kind of awkward. Thoughts?

Reply:
Indeed. The exporter should need to create an "AuxScene".